ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -295,31 +295,31 @@ Apply DataScienceCluster CustomResource
 Create DataScienceCluster CustomResource Using Test Variables
     [Documentation]
     [Arguments]    ${dsc_name}=${DSC_NAME}
-    ${file_path} =    Set Variable    tasks/Resources/Files/
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
     Copy File    source=${file_path}dsc_template.yml    destination=${file_path}dsc_apply.yml
-    Run    sed -i 's/<dsc_name>/${dsc_name}/' ${file_path}dsc_apply.yml
+    Run    sed -i ' ' -e 's/<dsc_name>/${dsc_name}/' ${file_path}dsc_apply.yml
     FOR    ${cmp}    IN    @{COMPONENT_LIST}
         IF    $cmp not in $COMPONENTS
-            Run    sed -i 's/<${cmp}_value>/Removed/' ${file_path}dsc_apply.yml
+            Run    sed -i ' ' -e 's/<${cmp}_value>/Removed/' ${file_path}dsc_apply.yml
```

Comment:
is it for fixing sed on MacOs?

Reply:
correct @bdattoma

Reply:
1. Such OS specific change shouldn't be part of such PR as this one is. It should be in a separate PR or commit at least. Otherwise someone (e.g. me) will put it back deliberately, because there is no explanation for this change and for regular Linux user it doesn't make sense.
2. If you do this change could you, please, check that this won't stop working to users with standard GNU sed? E.g. trying something similar, looks like it doesn't seem to work on my machine:
```
$ sed -i ' ' -e 's/my/Removed/' ahoj
sed: can't read  : No such file or directory
```
this seem to work on my machine:
```
$ sed -i'' -e 's/my/Removed/' ahoj
```

Reply:
@jstourac let me try your change. It should be compatible for Mac users too, if it passes CI/CD then it is suitable for Linux, but I need to be also able to execute on local

Reply:
it depends on the sed version. Kindly check the version

Reply:
could you let me know what version of sed you are using ?

Reply:
`$ sw_vers -productVersion
14.5`

Reply:
I think we should start limiting the usage of bash commands like sed where possible and where it doesn't complicate too much. 
We have `Create File From Template` keyword in RF which is comfortable for this kind of operations. Alternatively, we can use kustomize

Reply:
> could you let me know what version of sed you are using ?

It's not just about `sed` version but about the tool implementation... since MacOS was historically based on BSD, I believe that what Adrian is using is some kind of BSD sed implementation, whether on our Fedora machines we have a GNU sed implementation.

You can install GNU sed on MacOS too in case you want to keep this compatibility.

Reply:
@jstourac 
```
$ gsed --version
gsed (GNU sed) 4.9
```

Reply:
You mean you have this issue when using `gsed`? Or what does your comment mean?

Reply:
yes, it was not compatible with change it is

Reply:
Interesting, are you sure that this is the default sed that is used during the script execution? Please, be aware that script calls `sed` command, not `gsed`. Thus you need to have an alias for it or some other configuration so that `gsed` is used in the script.

Anyway, I don't want to spam this thread too much. We're slightly diverging here.